media/css/protocol/base/fonts/_firefox.scss

```diff
@@ -1,20 +1,19 @@
-// *** WARNING ***
-// The fonts used here are not approved for wide use on mozilla.org.
-// Don't reference this include anywhere but the "Fights for You" page.
-// Global brand fonts will be managed upstream in Protocol.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 
 @font-face {
-    font-family: 'Sharp Sans';
+    font-family: 'Firefox Sans';
     font-weight: normal;
     font-style: normal;
     font-display: swap;
     src: url('https://assets.mozilla.net/fonts/FirefoxSharpSans-Medium.woff2') format('woff2'),
          url('https://assets.mozilla.net/fonts/FirefoxSharpSans-Medium.woff') format('woff');
```

Comment:
Please remove all references to these fonts: https://github.com/mozmeao/infra/issues/1118#issuecomment-498386788

Reply:
Removed.

Reply:
Thanks @stephaniehobson